documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,166 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by VMware vSphere as source providers.
+
+[IMPORTANT]
+====
+Migration using Open Virtual Application (OVA) files as source providers is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
+ifdef::ostack[]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {ocp} web console.
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Adding a Red Hat {virt} source provider
+
+You can add a Red Hat {virt} source provider by using the {ocp} web console.
+
+endif::[]
+ifdef::cnv2[]
+= Adding {a-virt} destination provider
+
+You can add another {virt} destination provider to the {ocp} web console in addition to the default {virt} destination provider, which is the provider where you installed {project-short}.
```

Comment:
Two comments:
1. This sentence is relevant for an ocp provider, regardless if it's used as a source or as a destination. Currently it's mentioned only when ocp is added as a destination.
(please see my next comment on this  as well).

2. There is an option to add more than one "Host cluster" default ocp provider. We are not blocking it. So worth mentioning that the user can always leave the url and 'Service account bearer token' fields empty for adding more than one default ocp providers, while all are referenced to the same one.

Reply:
> 1. This sentence is relevant for an ocp provider, regardless if it's used as a source or as a destination. Currently it's mentioned only when ocp is added as a destination.

right, in MTV <= 2.3 the assumption was that providers are created in the openshift-mtv namespace (because that's what the UI allowed) and with the new UI, we rather expect providers to be defined in other namespaces - so adding the 'Host cluster' automatically is not that important anymore

@RichardHoch there are couple of things here:
1. mention that the same OpenShift Virtualization provider can be used as both source and destination provider
2. that specifically the automatically added 'Host cluster' can be used as both source and destination provider
3. mention that the migration can happen from the cluster that MTV is deployed on to another cluster and vice versa (from a remote cluster to the cluster that MTV is deployed on)

> 2. There is an option to add more than one "Host cluster" default ocp provider. We are not blocking it. So worth mentioning that the user can always leave the url and 'Service account bearer token' fields empty for adding more than one default ocp providers, while all are referenced to the same one.

we do it for the destination provider:
```
Service account token: Specify the cluster-admin service account token.

If both URL and Service account token are left blank, the local OpenShift cluster is used.
```
@RichardHoch that should appear in the section for source OpenShift Virtualization provider as well

Reply:
@ahadas The last comment is from the 2.4 documentation -- I am using it now as the definition for *Server account bearer token* for both OCP source and destination providers.